api/CanvasRenderingContext2D.json

```
@@ -2320,7 +2320,7 @@
                 "version_added": "24"
               },
               "safari": {
-                "version_added": false
+                "version_added": "7"
               },
               "safari_ios": {
                 "version_added": null
```

Hi @ramhugger. Based on the discussion on the previous PR, I think you could apply this change here too, then this PR could fully replace the previous one:

```suggestion
                "version_added": "7"
```